collector/manager/metrics_collector_manager.go

```diff
@@ -211,9 +224,9 @@ func (mcm *MetricsCollectorManager) declareMetricDefinitions(metricDetails []col
 
 		// put all the tags together for the full list of tags to be applied to this metric definition
 		allMetricTags := tags.Tags{}
-		allMetricTags.AppendTags(globalTags)   // global tags are overridden by...
+		allMetricTags.AppendTags(metricTags)   // metric tags are overriden by
 		allMetricTags.AppendTags(endpointTags) // endpoint tags which are overridden by...
-		allMetricTags.AppendTags(metricTags)   // metric tags.
+		allMetricTags.AppendTags(globalTags)   // global tags
```

Comment:
Aside from the global setting set by the admin. Should the endpoint tags overwrite the metric tags or vice versa?

I am thinking that maybe the metric ones overwriting the endpoint ones might make sense. But we don't want a developer to overwrite the global ones.